src/components/Navigation/README.md

```diff
@@ -173,7 +173,7 @@ Use to present a navigation menu in the [frame](https://polaris.shopify.com/comp
   <Navigation.Section
     items={[
       {
-        url: '/path/to/place',
+        url: '/',
```

Comment:
We always have one active items in the nav within the admin. This change reflects that int he basic example